nl.php

```diff
@@ -168,8 +168,8 @@
     // Habbo Ticker
     "Latest updates from Habbo Ticker" => "Nieuwste updates van Habbo Ticker",
     // This habbo added more than 5 new badges, 3 new groups and changed their look
-    "changed their motto" => "veranderde hun missie",
-    "changed their look" => "veranderde hun look",
+    "changed their motto" => "veranderde zijn/haar missie",
+    "changed their look" => "veranderde zijn/haar uiterlijk",
```

Comment:
Hebben we het in Habbo niet meestal over 'looks'?
```suggestion
    "changed their look" => "veranderde zijn/haar look",
```